packages/core/src/events/DefaultEventHandlers.ts

```diff
@@ -15,6 +17,41 @@ export class DefaultEventHandlers extends CoreEventHandlers {
   static draggedElementShadow: HTMLElement;
   static draggedElement: DraggedElement;
   static indicator: Indicator = null;
+  static lastDragPosition: { x: number; y: number } = null;
+
+  onDragOver(x: number, y: number, targetId: NodeId) {
+    const draggedElement = DefaultEventHandlers.draggedElement;
+    if (!draggedElement) {
+      return;
+    }
+
+    if (targetId) {
+      const node = this.store.query.node(targetId).get();
+      if (!node) {
+        return;
+      }
+    }
+
+    let node = (draggedElement as unknown) as Node;
+
+    if ((draggedElement as NodeTree).rootNodeId) {
+      const nodeTree = draggedElement as NodeTree;
+      node = nodeTree.nodes[nodeTree.rootNodeId];
+    }
+
+    const indicator = this.store.query.getDropPlaceholder(node, targetId, {
+      x,
+      y,
+    });
+
+    if (!indicator) {
+      return;
+    }
+    this.store.actions.setIndicator(indicator);
+    DefaultEventHandlers.indicator = indicator;
+  }
```

Comment:
This part seems to be the same as the `dragenter` handler used in the `drop` connector. 

Can we rename this function to something like `computeIndicator`, so we can use for both the `dragover` and `dragenter` handler?

Reply:
any update on this?